test/boost/tablets_test.cc

```diff
@@ -264,6 +264,10 @@ SEASTAR_TEST_CASE(test_tablet_metadata_persistence) {
 }
 
 SEASTAR_TEST_CASE(test_read_required_hosts) {
+    // FIXME: the test fails without using disable_consistent_topology_changes.
```

Comment:
reminder to open issue

Reply:
so this was using tablets **but not** consistent-topology? super weird

Reply:
hm, maybe enabling consistent topology causes load balancer to start running, and the load balancer preconditions are not satisfied because this test only partially updates the metadata, or something.

So potentially one way to fix it would be to disable the load balancer (IIRC we have some API for that)

Reply:
Yes, normally using tablets and not using raft-topology throws bad config. But this is thrown from the `main` function which is not executed in boost unit tests.